src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1593,17 +1594,14 @@ public void VerifyHMAC_HMACOutputLength_Invalid()
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/74115")]
         public void VerifyXmlResolver(bool provideResolver)
         {
-            HttpListener listener;
+            TcpListener listener;
             int port = 9000;
 
             while (true)
             {
-                listener = new HttpListener();
-                listener.Prefixes.Add($"http://127.0.0.1:{port}/");
-                listener.IgnoreWriteExceptions = true;
+                listener = new TcpListener(IPAddress.Loopback, port);
```

Comment:
We should remove this. This was done because `HttpListener` is .... stupid. 
Look at networking test: we should pass in 0 and Kernel will allocate unused port. 
It is atomic and save. We just need to read it back when the listener starts.